_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -1,51 +1,79 @@
 ---
 layout: default
-title: Supported log types
+title: Working with log types
 parent: Setting up Security Analytics
-nav_order: 16
+nav_order: 14
+redirect_from: 
+   - /security-analytics/sec-analytics-config/custom-log-type/
 ---
 
+# Working with log types
 
-# Supported log types
+Log types represent the different sources of data used for threat detection in Security Analytics. Log types are useful for categorizing or prepopulating [Detection rules]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/) when creating detectors from your source.
 
-Logs contain raw data about events that happen throughout a system and in its separate parts. As of OpenSearch 2.11, log types are grouped by category to help select, filter, and search the log types. 
+Log types come from the following sources:
```

Comment:
The list items do not describe sources but rather log types, so should this just say something like "Security Analytics supports the following log types:"?